test/cppsuite/configs/search_near_03_stress.txt

```diff
@@ -13,12 +13,14 @@ workload_generator=
     (
         collection_count=10,
         key_count_per_collection=2000,
-        key_size=30
+        key_size=30,
```

Comment:
Can you add a comment at the top explaining why such big values, i.e
`We want the key and values to take enough space to split across pages.`